apis/rust/node/src/node/mod.rs

```diff
@@ -157,10 +158,25 @@ impl DoraNode {
             drop_stream,
             cache: VecDeque::new(),
             dataflow_descriptor,
+            warned_unknown_output: BTreeSet::new(),
         };
         Ok((node, event_stream))
     }
 
+    fn validate_output(&mut self, output_id: &DataId) -> bool {
+        if !self.node_config.outputs.contains(output_id) {
+            if self.warned_unknown_output.contains(output_id) {
+                warn!("Output {output_id} is not part of the node's output list.");
```

Comment:
Regarding the wording of the warning: How about we mention that the output is ignored?

Reply:
Sure thing